checks/windows.py

```diff
@@ -210,9 +210,14 @@ def checkGameMode(lines):
 
 
 def checkWin10Hags(lines):
+    d3dAdapter = search('Loading up D3D11', lines)
+    hagsMessage = """The new Windows Hardware-accelerated GPU scheduling ("HAGS") added with Windows 10 version 2004 is currently known to cause performance and capture issues with OBS, games and overlay tools. It's a new and experimental feature and we recommend disabling it via <a href="ms-settings:display-advancedgraphics">this screen</a> or <a href="https://obsproject.com/wiki/How-to-disable-Windows-10-Hardware-GPU-Scheduler">these instructions</a>."""
```

Comment:
I'd rather not specify which version of Windows 10 doesn't work, as I'm quite sure 22H2 doesn't work either. Let's just specify WIndows 10 entirely.